src/Traits/FixturesTrait.php

```diff
@@ -99,7 +100,7 @@ public function getJsonFixture(string $fixtureName, $assoc = true)
 
     public function assertEqualsFixture(string $fixture, $data, bool $exportMode = false): void
     {
-        if ($exportMode || $this->globalExportMode) {
+        if ($exportMode || (isset($this->globalFixtureExport) && $this->globalExportMode)) {
```

Comment:
@KonstantinLapkovsky please check if this will works

```suggestion
        if ($globalExportMode ?? $exportMode) {
```